### PR TITLE
Update README.md with formatting guidelines

### PR DESCRIPTION
**Pull request by tiagobento**

### CODE REVIEW

**Review by ederign — APPROVED** [review]

**Comment by tiagobento** [comment]
cc @domhanak @tomasdavidorg @jstastny-cz @jomarko

**Comment by ederign** [comment]
@tiagobento let's work on making this verification automatic.

File: README.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ merge `{branch-name}` into master, and build them.
 In a few seconds you should see a new comment on the same issue, saying that a new build was triggered for you. The 
 GitHub Actions bot will also provide a link so you can follow the build logs and download artifacts, and another link to 
 access an Online Editor instance (2) containing your changes, once it's finished running. 
+- Please use Prettier to format the code before submitting a PR.
```
Comment (manstis) [inline README.md:137]: Did you know `kogito-apps` have a `github` hook to handle a bunch of things like this on commit?

https://github.com/kiegroup/kogito-apps/blob/master/ui-packages/package.json#L22

Use of the new "UI stack" (TS, React, lint, webpack etc) is increasing. IIUC legacy editors are to migrate (if not already doing so) and the number of developers in `kiegroup` using the new stack will grow. We had a hells own job trying to standardise dependencies and code style in the Java stack. IMO we're in muddy waters with non-standardisation within `kiegroup` for the new stack.

Personally, I think the respective repository owners of `kogito-tooling` and `kogito-apps` should agree and define the standards for everyone to use: `prettifier`, `[ts|es]lint`, dependency versions (where applicable, e.g. PF) and versioning (e.g. use of `^` versions) and hooks if it makes the developer experience consistent across repositories.

Has a conversation with the owner of `kogito-apps` happened regarding standardizations?

I half expect a "get lost Anstis" reply; but I'll keep pushing for alignment.

**Comment by jomarko** [comment]
https://github.com/kiegroup/kogito-tooling/pull/450#issuecomment-799702598 - @ederign and lets make it via jira shared with everyone please

File: README.md (hunk 1 above)
Comment (domhanak) [reply README.md:137]: I think this matches your proposal @manstis  - https://issues.redhat.com/browse/KOGITO-3560 - let's use it to track this effort.

**Pull request merged**